src/retrieval_service/app/__init__.py

```diff
@@ -11,5 +11,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
```

Comment:
Based on previous, at a guess this is what was meant?  (Based on a similar pattern in src/retrieval_service/datastore/__init__.py, datastore/providers/__init__.py)
```suggestion
# limitations under the License.

from .app import EMBEDDING_MODEL_NAME, init_app, parse_config

__ALL__ = [EMBEDDING_MODEL_NAME, init_app, parse_config]
```